### ISSUE TITLE
Visual Studio Mac extensions mpack  Installation failure

### ISSUE THREAD

**Issue by kalirajanganesan** [mhutch/MonoDevelop.AddinMaker#76]
I have created the Visual studio mac extensions mpack file using ide-extensions project after installing addin.maker extesnions. I am facing below installation warning while installing my mpack

Error:

The add-in Mytemplate could not be updated because some of its dependencies missing or not compatible

required:Mono.Develop.Core 8.2.5 , found : MonoDevelop.Core 8.1.2 required:Mono.Develop.Ide 8.2.5 , found : MonoDevelop.Ide8.1.2

**Comment by kalirajanganesan** [comment]
@mrward  @mhutch  Couldd you please help to resolve that issue. I get some reference like changes the mono developer addin nuget packages like that and tried it was not working. Could you please share exact information to how to made my mpack file support for all versions of monodevelop.Ide and Monodevelop.core

**Comment by mrward** [comment]
The simplest way I think would be to build your addin with the lowest version of VS Mac you want to support.

Not sure if there is a way to specify an older version when using addin maker and building an addin using a later VS Mac version.

**Comment by mhutch** [comment]
Yes, it's possible to explicitly override the target version https://github.com/mhutch/MonoDevelop.AddinMaker/wiki/Build-System#mdcoreversionoverride
